tickets/forms.py

```diff
@@ -70,6 +71,13 @@ def __init__(self, *args, **kwargs):
                        'Return',
                        'Whitelist']]
 
+IS_EXPLICIT_KEEP_USER_PREFERENCE_KEY = "K"
+IS_EXPLICIT_ADD_FLAG_KEY = "A"
+IS_EXPLICIT_REMOVE_FLAG_KEY = "R"
+IS_EXPLICIT_FLAG_CHOICES = ((IS_EXPLICIT_KEEP_USER_PREFERENCE_KEY, "Keep user preference"),
+                            (IS_EXPLICIT_ADD_FLAG_KEY, "Add \"is explicit\" flag"),
```

Comment:
It'd be easier here to just use `"""` or `'` as string quotes so that you don't need to escape